undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd

```diff
@@ -100,6 +100,7 @@
                 <xs:attribute name="allow-equals-in-cookie-value" type="xs:boolean" default="false"/>
                 <xs:attribute name="no-request-timeout" type="xs:int"/>
                 <xs:attribute name="request-parse-timeout" type="xs:int"/>
+                <xs:attribute name="disallowed-methods" type="xs:string"/>
```

Comment:
should be type="stringList" see https://github.com/wildfly/wildfly/pull/7440/files#diff-1b2f64905e4ae90d9e6190931b526fe3R372 as example